frontend/json-schema/authorship-schema.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I'm not sure if non-required fields need to be included here, but there is also a field `isBinary` (and `isIgnored` added in #1647) that are only serialized if true.

See: [FileResult.java](https://github.com/reposense/RepoSense/blob/2b864022bbf5db55ae9d4f96eaf0a70e961646f5/src/main/java/reposense/authorship/model/FileResult.java)